README.md

```diff
@@ -1,11 +1,19 @@
-# Contract level buyer information
+# Multiple buyers information
 
-The core OCDS schema provides space for a single `buyer` to be described for each contracting process. The buyer is defined as the organizations whose funds are directly used for the purchase of the goods, works or services described in the contract.
+The core OCDS schema provides space for a single `buyer` to be described for each contracting process. The buyer is defined as the organization whose budget will be used for the purchase of the goods, services or works described in the contract.
 
-However, some forms of contracting process, such as framework agreements, may result in multiple contracts, with each contract signed by a different buyer. This extension adds a `contracts.buyer` object to support this scenario.
+However, some forms of contracting process, such as framework agreements, may result in multiple awards or contracts with a different buyer or buyers for each. This extension adds `awards.buyers` and `contracts.buyer` objects to support these scenarios.
+
+## Legal Context
+
+In the European Union, this extension's fields correspond to [eForms OPT-300-Contract-Signatory](https://docs.ted.europa.eu/eforms/latest/reference/business-terms/). See [OCDS for eForms](https://standard.open-contracting.org/profiles/eforms/latest/en/) for correspondences to eForms fields.
 
 ## Example
 
+### Contracts
+
+2 contracts are issued with a different buyer for each.
```

Comment:
I don't think it's written down anywhere, but I think we generally write smaller numbers as words.

```suggestion
Two contracts are issued with a different buyer for each.
```